src/Quasar/Mount/Couchbase.purs

```diff
@@ -57,26 +64,37 @@ fromJSON
   <=< decodeJson
 
 toURI ∷ Config → URI.AbsoluteURI
-toURI { host, user, password } =
+toURI { host, bucketName, password, docTypeKey, queryTimeout } =
   URI.AbsoluteURI
     (Just uriScheme)
-    (URI.HierarchicalPart (Just (URI.Authority Nothing (pure host))) Nothing)
+    (URI.HierarchicalPart
+      (Just (URI.Authority Nothing (pure host)))
+      (Just (Right (P.rootDir </> P.file bucketName))))
     (Just (URI.Query props))
   where
   props ∷ L.List (Tuple String (Maybe String))
   props = L.Nil
-    <> maybe L.Nil (\u -> pure $ Tuple "username" (Just u)) user
-    <> maybe L.Nil (\p -> pure $ Tuple "password" (Just p)) password
+    <> pure (Tuple "password" (Just password))
+    <> pure (Tuple "docTypeKey" (Just docTypeKey))
+    <> maybe L.Nil (pure <<< Tuple "queryTimeoutSeconds" <<< Just <<< show <<< unwrap) queryTimeout
```

Comment:
You're right in that regard, I just don't think that it's likely that `queryTimeout` will ever change from `Maybe Seconds` again. It may change name, be altered in some other way in the config, but `Seconds` seems like a safe bet.